### ISSUE TITLE
curl_exec calls don't work after connecting to a VPN

### ISSUE THREAD

**Issue by TerrorSquad** [markshust/docker-magento#703]
**Description**
I'm working on a Dutch website and from my local, I need to connect to a VPN  in order to fetch data such as product prices from an ERP. In the checkout, we're using the `tig/postcode-magento2` package to fetch address information and validate whether the entered address exists. The module exposes the **/postcode/address/service** controller that receives postcode and house number as query parameters. To send the actual request to the 3rd party API the following method is used inside the tig module: **\Magento\Framework\HTTP\ZendClient::request**. That function call will later on end up calling  **\Magento\Framework\HTTP\Adapter\Curl::read** - this method contains the **curl_exec** call.

If I don't connect to the VPN, the endpoint will return data about the validity of the entered postcode/house number combo.
If I do connect to the VPN before or after starting the containers, the **curl_exec** call will simply return **false**, which will later throw an exception because it cannot be parsed as a valid response.

Postcode validation is an integral part of the checkout validation. I know I can work around this issue by modifying the code inside the **vendor** directory to return a successful response with some dummy data, but I'd rather have a proper solution.

I would hope that this is something that can be solved by modifying the docker-compose.yml file.

**Steps To Reproduce**
- Enable a VPN connection
- Execute code that calls **\Magento\Framework\HTTP\ZendClient::request** 

**Expected Result**
Response is returned

**Actual Result**
**false** is returned

**Comment by markshust** [comment]
I don't believe this is any issue with this project, as this simply exposes the services through ports. It shouldn't have anything to do with you running a VPN or not. I'm going to close this out as I cannot confirm this is an issue.
